src/space/objects/realization/blackhole.js

```diff
@@ -0,0 +1,78 @@
+import Vector from "../../utils/vector";
+
+export default class BlackHole(){
+
+
+  /**
+    * Constructsa black hole game element
+    *
+    * @param {Integer} x - the x position of black hole in the map
+    * @param {Integer} y - the y position of black hole in the map
+    * @reference Black Hole (Warp) by Timothy Courtney (timothycourtney.io)
+    *  Licensed under Creative Commons: By Attribution 3.0 License
+    *  http://creativecommons.org/licenses/by/3.0/
+    */
+  constructor(x,y){
```

Comment:
in the game, I think a black hole won't move at all, the same as the loop hole. If I use vector, it will be {x:0, y:0}.
OR you want to me make it as a moveable black hole?

Reply:
It doesn't really matter, whether the black hole will move or not - the whole game is using vectors. If you don't want the black hole to move, do not set it's velocity, since it defaults to (0, 0). 

Whether the black hole will move or not, that decision I will leave to you ;)